streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableValueGetterSupplier.java

```diff
@@ -21,4 +21,5 @@
 
     KTableValueGetter<K, V> get();
 
+    String[] storeNames();
```

Comment:
As mentioned above, why not `List<String>` or `Collection<String>`? It would save some of the work being done in the above methods, i.e., copying to a List so it can be converted back into an array.